nemo/core/optim/lr_scheduler.py

```diff
@@ -434,17 +430,32 @@ def prepare_lr_scheduler(
         logging.info('Scheduler not initialized as no `sched` config supplied to setup_optimizer()')
         return None
 
-    # Get name of the scheduler
-    scheduler_name = scheduler_config['name']
-
     # Try instantiation of scheduler params from config class path
     try:
-        scheduler_conf = hydra.utils.instantiate(scheduler_args)
```

Comment:
You replace one line of instantiate with many lines which are more complicated.
I am curious as to the motivation here.